crates/router/src/connector/forte.rs

```diff
@@ -0,0 +1,468 @@
+mod transformers;
+
+use std::fmt::Debug;
+use error_stack::{ResultExt, IntoReport};
+
+use crate::{
+    configs::settings,
+    utils::{self, BytesExt},
+    core::{
+        errors::{self, CustomResult},
+        payments,
+    },
+    headers, logger, services::{self, ConnectorIntegration},
+    types::{
+        self,
+        api::{self, ConnectorCommon, ConnectorCommonExt},
+        ErrorResponse, Response,
+    }
+};
+
+
+use transformers as forte;
+
+#[derive(Debug, Clone)]
+pub struct Forte;
+
+impl<Flow, Request, Response> ConnectorCommonExt<Flow, Request, Response> for Forte 
+where
+    Self: ConnectorIntegration<Flow, Request, Response>,{
+    fn build_headers(
+        &self,
+        req: &types::RouterData<Flow, Request, Response>,
+        _connectors: &settings::Connectors,
+    ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
+        let mut headers = vec![
+            (
+                "X-Forte-Auth-Organization-Id".to_string(),
+                "org_436915".to_string(),
```

Comment:
do not hardcode this value, fetch this from connector_meta that you provided while creating the `payment connector` metadata.